tango/step_graph.py

```diff
@@ -67,6 +68,11 @@ def __init__(self, params: Dict[str, Params]):
         del done
         del todo
 
+        # make sure default steps are available
+        import tango.steps  # noqa: F401
+
+        import_integrations()
```

Comment:
Why did you add this? I'm not a huge fan of unnecessary imports. That's one of my biggest annoyances with AllenNLP.

Reply:
Because when you check out https://github.com/allenai/tango-example, and you try to run `tango anything`, it's completely useless. It has the `print` step and nothing else.

Reply:
I'm not sure what you mean. I had no issues running that without these lines. Tango already lazily imports steps and integrations automatically when needed.

Reply:
I have latest tango main, and latest tango-example. In the root directory of -example, I run `tango run euler.jsonnet` without the `--import-package`. I'm only doing this so I get a list of available steps.

It prints
```
tango.common.exceptions.ConfigurationError: 'cmul' not in acceptable choices for type: ['dataset_remix', 'print']
```

I see I have none of the integrations available. Only two steps here.

Reply:
They are available, that's just a bad error message.

Reply:
I can fix that in a separate PR

Reply:
Not that I don't believe you, but how?

Reply:
https://github.com/allenai/tango/pull/109 fixes.

> Not that I don't believe you, but how?

This function is used to guess which modules need to be imported:
https://github.com/allenai/tango/blob/b2781fd5b410c08546e64919a4228f5ffc365dfa/tango/common/registrable.py#L159

Reply:
Other than this the PR LGTM

Reply:
I reverted those changes.